elrond-wasm/src/types/managed/big_uint_operators.rs

```diff
@@ -62,7 +62,7 @@ macro_rules! binary_operator {
             type Output = BigUint<M>;
 
             fn $method(self, other: u64) -> BigUint<M> {
-                let other_handle = M::managed_type_impl().bi_new(other as i64);
+                let other_handle = M::managed_type_impl().bi_new_handle(other as i64);
```

Comment:
Same.
We need to use something based on `bigIntSetInt64` here.
One of the temporary handles can be used.